src/isomorphic/classic/element/ReactElementType.js

```diff
@@ -23,7 +23,7 @@ export type ReactElement = {
   $$typeof: any,
   type: any,
   key: any,
-  ref: any,
+  ref: string | (elem: ?HTMLElement) => void,
```

Comment:
flow coverage --color
On Mon, 29 Aug 2016 at 19:04, Christopher Chedeau <notifications@github.com>
wrote:

> In src/isomorphic/classic/element/ReactElementType.js
> <https://github.com/facebook/react/pull/7600#discussion_r76634323>:
>
> > @@ -23,7 +23,7 @@ export type ReactElement = {
> >    $$typeof: any,
> >    type: any,
> >    key: any,
> > -  ref: any,
> > +  ref: string | (elem: ?HTMLElement) => void,
>
> cc @thejameskyle <https://github.com/thejameskyle>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/react/pull/7600/files/6a1166f44d77d267879a9e39b6cde9241a569043#r76634323>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAB3gk_ZuNizzgRHP7SeOrxVSKUMDotlks5qkwMFgaJpZM4JvD5_>
> .
>